installer/local_docker/tasks/main.yml

```diff
@@ -101,7 +101,7 @@
     state: started
     image: postgres:9.6
     volumes:
-      - "{{ postgres_data_dir }}:/var/lib/postgresql/data"
+      - "{{ postgres_data_dir }}:/var/lib/postgresql/data:Z"
```

Comment:
does this work generally on all platforms?

Reply:
@matburt I've tested successfully on **CentOS 7.x** and **Ubuntu 16.04 LTS**. I think these are fairly representative of RHEL based and Debian based distros.